src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyleBindableProperty.cs

```diff
@@ -49,6 +49,14 @@ public partial class ViewStyle
             defaultValueCreator: (bindable) => ((ViewStyle)bindable).focusable
         );
 
+        /// <summary> Bindable property of FocusableChildren. Please do not open it. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty FocusableChildrenProperty = BindableProperty.Create(nameof(FocusableChildren), typeof(bool?), typeof(ViewStyle), null,
```

Comment:
I am not sure but in the comment, I could see the default value is `true`, but here default value looks `null`.
could you check?